src/Illuminate/Validation/Rules/Password.php

```diff
@@ -361,4 +371,27 @@ protected function fail($messages)
 
         return false;
     }
+
+    /**
+     * Returns the translated error message.
+     *
+     * @param  string  $key
+     * @return string
+     */
+    protected function getErrorMessage($key)
+    {
+        $messages = [
+            'validation.password.mixedCase' => 'The :attribute must contain at least one uppercase and one lowercase letter.',
+            'validation.password.letters' => 'The :attribute must contain at least one letter.',
+            'validation.password.symbols' => 'The :attribute must contain at least one symbol.',
+            'validation.password.numbers' => 'The :attribute must contain at least one number.',
+            'validation.password.uncompromised' => 'The given :attribute has appeared in a data leak. Please choose a different :attribute.',
+        ];
+
+        if (($message = $this->validator->getTranslator()->get($key)) !== $key) {
+            return $message;
+        }
+
+        return $messages[$key];
```

Comment:
What about directly return instead of defining $messages variable on the top of this method?
```php
return [
    'validation.password.mixedCase' => 'The :attribute must contain at least one uppercase and one lowercase letter.',
    'validation.password.letters' => 'The :attribute must contain at least one letter.',
    'validation.password.symbols' => 'The :attribute must contain at least one symbol.',
    'validation.password.numbers' => 'The :attribute must contain at least one number.',
    'validation.password.uncompromised' => 'The given :attribute has appeared in a data leak. Please choose a different :attribute.',
][$key];
```